hack/build-coredns.sh

```diff
@@ -4,16 +4,22 @@ set -euo pipefail
 
 PLUGIN_PATH=$(readlink -f "$(dirname "$0")/..")
 
-# Create a temporary directory for cloning openshift/coredns repo.
+# Create a temporary directory for cloning coredns repo.
 # The directory will be deleted after the execution of script.
 BASE_PATH=$(mktemp -d)
 trap 'chmod -R u+w "${BASE_PATH}"; rm -rf "${BASE_PATH}"' EXIT
 
 cd "${BASE_PATH}"
-# Clone openshift/coredns repo.
-git clone https://github.com/openshift/coredns
-
-# Add the "ocp_dnsnameresolver" plugin to the cloned openshift/coredns repo.
+if [ ! -z "${1-}" ] && [ "${1}" = "upstream" ]
+then
+    echo "Cloning coredns/coredns repo"
+    git clone https://github.com/coredns/coredns
+else
+    echo "Cloning openshift/coredns repo"
+    git clone https://github.com/openshift/coredns
+fi
```

Comment:
```suggestion
COREDNS_URL="https://github.com/openshift/coredns"
if [ "${1}" = "upstream" ]; then
    CLONE_URL="https://github.com/coredns/coredns"
done
echo "Cloning from ${COREDNS_URL}"
git clone "${COREDNS_URL}"
```

Reply:
Made the required changes to the script.